### ISSUE TITLE
Mini-Kinetics pseudo scene labels

### ISSUE THREAD

**Issue by Fazlik995** [vt-vl-lab/SDN#4]
@jinwchoi 

In the paper, you did write "we obtain a pseudo scene label p˜ ∈ P˜ by running Places365
dataset pre-trained ResNet-50 [73] on the Kinetics dataset".

Since you pre-trained models on Mini-Kinetic dataset, why you obtain pseudo scene labels of Kinetics dataset, not Mini-Kinetics dataset? (I guess in your experiment, pseudo scene labels of Kinetics dataset is not needed)

Thank you!

**Comment by jinwchoi** [comment]
Mini-Kinetics is a subset of the full Kinetics.

**Comment by jinwchoi** [comment]
Your recent issues are not really code issues. The purpose of "Issues" of github is reporting code issues. For the questions on the methods and/or details, feel free to ask me via email. Thanks.

**Comment by Fazlik995** [comment]
Sorry. I will ask further questions via email.

Thank you.
